src/mobile/components/layout/stylesheets/buttons.styl

```diff
@@ -112,8 +113,7 @@ a.nav-image-item
 
 // Avant Garde rectangular buttons
 .avant-garde-box-button
-  avant-garde()
-  font-size avant-garde-font-size
+  text('mediumText')
```

Comment:
This remains named `avant-garde-foo-bar` because so many other related styles follow the same pattern 😫 and it seem unwieldy to change them all. 

But if anyone is offended by that I can dig deeper on this.

(This is the change that is expected to ripple widely — see [Slack thread](https://artsy.slack.com/archives/C9XJKPY9W/p1600274937047200))

Reply:
Yeah, I'm doing the same for a lot of things (homepage touches a ton of global stuff) 👍